Scripts/Core/EditorVR.MiniWorlds.cs

```diff
@@ -137,10 +137,10 @@ internal void UpdateMiniWorlds(ConsumeControlDelegate consumeControl)
 					miniWorldRayOrigin.rotation = referenceTransform.rotation * Quaternion.Inverse(miniWorld.miniWorldTransform.rotation) * originalRayOrigin.rotation;
 					miniWorldRayOrigin.localScale = Vector3.Scale(inverseScale, referenceTransform.localScale);
 
-					var directSelection = evr.m_DirectSelection;
+					var directSelection = evr.GetNestedModule<DirectSelection>();
```

Comment:
I'm not sure this is necessary any more? From what I can tell we are just checking for `!= null`.